From e0b2222000d04b43e7892148dc9892c9375de939 Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Sat, 5 May 2012 15:04:24 -0300
Subject: [PATCH] mxs-duart: use ttyAMA for device name

This allow to easy configuration of u-boot avoiding divergences
between Freescale's and mainline kernels.

Upstream-Status: Inapropriate [configuration]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>
---
 drivers/serial/mxs-duart.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/serial/mxs-duart.c b/drivers/serial/mxs-duart.c
index 64e5057..2ba0c33 100644
--- a/drivers/serial/mxs-duart.c
+++ b/drivers/serial/mxs-duart.c
@@ -665,7 +665,7 @@ static int __init duart_console_setup(struct console *co, char *options)
 }
 
 static struct console duart_console = {
-	.name = "ttyAM",
+	.name = "ttyAMA",
 	.write = duart_console_write,
 	.device = uart_console_device,
 	.setup = duart_console_setup,
@@ -688,8 +688,8 @@ console_initcall(duart_console_init);
 
 static struct uart_driver duart_drv = {
 	.owner = THIS_MODULE,
-	.driver_name = "ttyAM",
-	.dev_name = "ttyAM",
+	.driver_name = "ttyAMA",
+	.dev_name = "ttyAMA",
 	.major = SERIAL_DUART_MAJOR,
 	.minor = SERIAL_DUART_MINOR,
 	.nr = 1,
-- 
1.7.10

